tests/integration/cli/test_cli_entrypoint.py

```
@@ -0,0 +1,28 @@
+        env_path,
+        "--model-config",
+        model_path,
+        "--trainer-config",
+        trainer_path,
+    ]
+    with (
+        patch.object(sys, "argv", test_args),
+        patch("src.main.Trainer") as mock_trainer_class,
+        patch("src.main.main") as mock_main,
+    ):
+        exec(open("src/main.py").read())
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)